syllabuspdf/apirequest.py

```diff
@@ -32,6 +38,7 @@ def fetch_allevents(id):
 	'''Fetches assignments and events for course with given id'''
 	if oauthtoken is None:
 		raise Exception("Invalid Authorization Token")
+		logger.warning("Missing oauthtoken")
```

Comment:
This conditional is repeated in each api function. Can we turn this into a function that is called from the api functions?